### PR TITLE
REFERENCE and README changes

### PR DESCRIPTION
**Pull request by timidri**
- add Puppet and Dell Contact info to README
- minor README changes
- fix summary line for types in REFERENCE

### CODE REVIEW

**Comment by nigelkersten** [comment]
Readme looks good. 

Should the reference lines be cut off partway through sentences?

**Comment by timidri** [comment]
@nigelkersten The issue is that the summary line cannot be longer than 140 characters. I had a choice to either remove that altogether or keep the cut off version for now leaving "prettifying" the cutoff for a future PR. We could cut off after the first sentence and add something like an ellipsis (...). When you click on a type name, you go to the full documentation where the same information is present without a cutoff.

Shall we keep it like this or remove the summary from the overview of types? Note that the same happens with tasks a bit down REFERENCE.md.

**Comment by nigelkersten** [comment]
Ah. Yes lets keep it like this and we can prettify on sentence boundaries later.

**Comment by ameer-jabbar** [comment]
Quick question: these changes would require a new release? If so, I was wondering if there could be a small change updated with this. We wanted to have the 'Application-Type' HTTP header sent in each REST call that the module makes. If it make sense to do it in other PR, please go ahead with that instead. This was the change that I was thinking about in the lib/puppet/transport/powerstore.rb file:

```
def call_op(path_params, query_params, header_params, body_params, operation_path, operation_verb, parent_consumes)
      uri_string = "#{connection_info[:schema]}://#{connection_info[:host]}:#{connection_info[:port]}#{connection_info[:base_path]}#{operation_path}" % path_params
      if !query_params['query_string'].nil?
        uri_string = uri_string + '?' + query_params['query_string']
      else
        if operation_verb == 'Get' # && operation_path.include?('query')
          query_params['select'] = '*'
        end
        uri_string = uri_string + '?' + to_query(query_params) unless query_params.empty?
      end
++      # FIXME: Obtain the version value for 'Application-Type' dynamically (at least, upon first initialization)
++      header_params['Application-Type'] = "Puppet Module for PowerStore/0.8"
      header_params['Content-Type'] = parent_consumes
```

**Comment by timidri** [comment]
@ameer-jabbar This needs to be changed in the generation template. I will do that, run a test suite and submit a new PR specifically for that change.

**Comment by ameer-jabbar** [comment]
> @ameer-jabbar This needs to be changed in the generation template. I will do that, run a test suite and submit a new PR specifically for that change.

Sounds great, thank you!

File: README.md [hunk 1]
```diff
@@ -416,24 +435,27 @@ powerstore_email_notify_destination
...
+```log
 pdk bundle exec rake strings:generate:reference
 ```
 
-## Contributors
-
 ## Contact
 
-... forthcoming ...
+... Dell contacts here ...
```
Comment (AronAtDell) [inline README.md:453]: Could we add the contact information here:

For Puppet module issues, question or feedback <fill in the blanks>.

Dell EMC does not provide support for any source code modifications. For any Dell EMC PowerStore issues, questions or feedback, please contact support  https://www.dell.com/support/

Comment (timidri) [reply README.md]: @AronAtDell Added these changes to the PR.

Comment (timidri) [reply README.md]: @AronAtDell Can you please approve the PR? Thanks!

**Pull request merged**